.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Description
```

Comment:
We should split the description up into 2 fields: Current Behavior and Expected Behavior